proxmox/session.go

```diff
@@ -167,6 +167,11 @@ func (s *Session) SetAPIToken(userID, token string) {
 	s.AuthToken = auth
 }
 
+func (s *Session) SetTicket(ticket, csrfPreventionToken string) {
```

Comment:
Can you make this private so `setTicket`? Nothing in `Session` should be exposed.

Reply:
func (Session)setTicket made private